verilog/analysis/verilog_linter.cc

```diff
@@ -62,7 +62,7 @@ ABSL_FLAG(verilog::RuleBundle, rules, {},
 ABSL_FLAG(std::string, rules_config, "",
           "Path to lint rules configuration file. "
           "Disables --rule_config_search if set.");
-ABSL_FLAG(bool, rules_config_search, false,
+ABSL_FLAG(bool, rules_config_search, true,
```

Comment:
We can't switch this on by default, as this also affects the stand-alone tool where the default expectation currently is that this is not searching for such rules.
(we can probably change that expectation long-term, but we should not introduce a behavioral change in passing).

If you want to force this flag on in the language server (which is more reasonable), there is `absl::SetFlag()`...

Reply:
ok, will fix it.

Reply:
I declared the flag for `--rules-config-search` and set it to true for `VerilogLanguageServer`. We can also have it disabled by default and tell users to set this flag in their configuration of language server